hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheCreateConfigOperation.java

```diff
@@ -86,7 +86,7 @@ public void run() throws Exception {
         if (!ignoreLocal) {
             response = service.createCacheConfigIfAbsent(config);
         }
-        if (createAlsoOnOthers && response == null) {
+        if (createAlsoOnOthers) {
```

Comment:
Can you simplify this code. It contains a lot of nested control structures. This can easily be fixed by pulling out some methods.